zerver/management/commands/runtornado.py

```diff
@@ -102,7 +102,7 @@ def inner_run():
             translation.activate(settings.LANGUAGE_CODE)
 
             print("Validating Django models.py...")
-            self.validate(display_num_errors=True)
+            self.check(display_num_errors=True)
```

Comment:
For this one, can you include in the commit message a link to where in the Django release notes it explains this change?